File: SampleApp/java/SDK/app/src/main/java/jp/fluct/sample/samplefluctsdkapp/BannerActivity.java
package jp.fluct.sample.samplefluctsdkapp;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

import jp.fluct.sample.samplefluctsdkapp.banner.BannerLayoutXmlActivity;
import jp.fluct.sample.samplefluctsdkapp.banner.BannerRecyclerActivity;

public class BannerActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.banner_activity);

        this.<Button>findViewById(R.id.impl_on_xml).setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {
                startActivity(
                        new Intent(BannerActivity.this, BannerLayoutXmlActivity.class)
                );
            }

        });

        this.<Button>findViewById(R.id.recycler).setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {
                startActivity(
                        new Intent(BannerActivity.this, BannerRecyclerActivity.class)
                );
            }

        });
    }
}

File: SampleApp/kotlin/app/src/main/java/jp/fluct/sample/kotlinapp/BannerActivity.kt
package jp.fluct.sample.kotlinapp

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import jp.fluct.sample.kotlinapp.banner.BannerLayoutXmlActivity
import jp.fluct.sample.kotlinapp.banner.BannerRecyclerActivity

class BannerActivity : AppCompatActivity() {

    private val implOnXml by lazy { findViewById<Button>(R.id.impl_on_xml) }
    private val implOnRecycler by lazy { findViewById<Button>(R.id.impl_on_recycler) }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.banner_activity)

        implOnXml.setOnClickListener {
            startActivity(Intent(this, BannerLayoutXmlActivity::class.java))
        }

        implOnRecycler.setOnClickListener {
            startActivity(Intent(this, BannerRecyclerActivity::class.java))
        }
    }

}
